.gitlab-ci.yml

```
@@ -22,6 +22,7 @@ stages:
   BUILD_OMP: "OFF"
   BUILD_CUDA: "OFF"
   BUILD_HIP: "OFF"
+  CONFIG_LOG: "1"
```

nit: this looks slightly inconsistent to me, moving to BOOL would allow
```suggestion
  CONFIG_LOG: "ON"
```